code/obj/item/scripture.dm

```diff
@@ -0,0 +1,120 @@
+//Place for the new chaplain books to live since they function differently from the bible
+
+/obj/item/scripture
+	name = "blank scripture"
+	icon = 'icons/obj/writing.dmi'
+	icon_state = "blankhb"
+	inhand_image_icon = 'icons/mob/inhand/hand_books.dmi'
+	item_state = "blankhb"
+	desc = "An interpretive holy scripture touting… just about whatever you want, really."
+	var/item_state_base = "blankhb"
+	var/open = FALSE
+
+	attack_self(mob/user)
+		if (user.find_in_hand(src))
+			if (!src.open)
+				src.open(user)
+			else
+				src.close(user)
+			user.update_inhands()
+
+	proc/open (mob/user as mob)
+		src.open = TRUE
+		icon_state = "[icon_state]open"
+		item_state = "[item_state]open"
```

Comment:
I don't like a var having the same name as a proc.

Maybe call it is_open

Reply:
Should be changed, with thanks to gallowscalibrator